marathon/src/test/scala/io/buoyant/marathon/v2/AppIdNamerTest.scala

```diff
@@ -11,6 +11,7 @@ class AppIdNamerTest extends FunSuite with Awaits {
   val ttl = 10.millis
   val err = ChannelWriteException(null)
   val exc = Future.exception(err)
+  val timer = new MockTimer
```

Comment:
you may not want to share this across tests (i don't know how stateful it is).  probably better as `def newTimer`